src/main/java/gov/cms/ab2d/worker/processor/MockEventClient.java

```diff
@@ -0,0 +1,11 @@
+package gov.cms.ab2d.worker.processor;
+
+import gov.cms.ab2d.eventclient.clients.EventClient;
+import gov.cms.ab2d.eventclient.events.LoggableEvent;
+
+public class MockEventClient implements EventClient {
```

Comment:
Maybe use beans to overwrite events in testing.

Reply:
Integrating with EventClient is another story entirely.  This mock is just a placeholder.